controlplane/kubeadm/internal/workload_cluster_test.go

```diff
@@ -1260,6 +1262,69 @@ func TestClusterStatus(t *testing.T) {
 	}
 }
 
+func TestUpdateFeatureGatesInKubeadmConfigMap(t *testing.T) {
+	tests := []struct {
+		name                     string
+		clusterConfigurationData string
+		newFeatureGates          map[string]bool
+		wantFeatureGates         map[string]bool
+	}{
+		{
+			name: "it update feature gates",
```

Comment:
```suggestion
			name: "update feature gates",
```

Reply:
missed the s there, `how about it updates feature gates`, since it matches the other tests?